AtCoder's submission result pages are updated

## Description

The submission result page https://atcoder.jp/contests/abc168/submissions/13347505 seems to be updated.
This breaks our code.

This doesn't affect most users, but at least breaks our CI.
Remove the feature to retrieve the detailed info from submissions?

## Error log

https://github.com/online-judge-tools/api-client/runs/1178238041

```
=========================== short test summary info ============================
FAILED tests/get_problem_topcoder.py::GetProblemTopcoderTest::test_10760 - As...
FAILED tests/dispatch.py::DispatchAtCoderTest::test_submission_from_url - Ass...
FAILED tests/type.py::TypeTest::test_download_contest - AssertionError
FAILED tests/dispatch.py::DispatchAtCoderTest::test_submission_from_url - Ass...
FAILED tests/service_atcoder.py::AtCoderContestTest::test_iterate_submissions
FAILED tests/service_atcoder.py::AtCoderProblemTest::test_iterate_submissions
FAILED tests/service_atcoder.py::AtCoderSubmissionTest::test_get_score_float
FAILED tests/service_atcoder.py::AtCoderSubmissionTest::test_get_source_code
FAILED tests/service_atcoder.py::AtCoderSubmissionTest::test_get_test_cases
FAILED tests/service_atcoder.py::AtCoderSubmissionTest::test_get_test_sets - ...
FAILED tests/service_atcoder.py::AtCoderSubmissionTest::test_submission_info
FAILED tests/service_atcoder.py::AtCoderSubmissionTest::test_submission_info_compile_warnings
= 12 failed, 142 passed, 5 skipped, 2 xfailed, 108 warnings in 149.93s (0:02:29) =
```

``` python
__________________ AtCoderSubmissionTest.test_submission_info __________________

self = <tests.service_atcoder.AtCoderSubmissionTest testMethod=test_submission_info>

    def test_submission_info(self):
        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/agc030/submissions/3904911')
>       data = submission.download_data()

tests/service_atcoder.py:209: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = AtCoderSubmission.from_url('https://atcoder.jp/contests/agc030/submissions/3904911')

    def download_data(self, *, session: Optional[requests.Session] = None) -> AtCoderSubmissionDetailedData:
        """
        :note: `Exec Time` is undefined when the status is `RE` or `TLE`
        :note: `Memory` is undefined when the status is `RE` or `TLE`
        """
        session = session or utils.get_default_session()
        resp = _request('GET', self.get_url(type='beta', lang='en'), session=session)
        soup = bs4.BeautifulSoup(resp.content.decode(resp.encoding), utils.html_parser)
        timestamp = datetime.datetime.now(datetime.timezone.utc).astimezone()
    
        # Submission #N
        id_, = soup.find_all('span', class_='h2')
        assert id_.text == 'Submission #{}'.format(self.submission_id)
    
        # Source Code
        source_code = soup.find(id='submission-code')
        source_code = source_code.text.encode()
    
        # get tables
        tables = soup.find_all('table')
        if len(tables) == 3:
            submission_info, test_cases_summary, test_cases_data = tables
        elif len(tables) == 1:
            submission_info, = tables
            test_cases_summary = None
            test_cases_data = None
        else:
>           assert False
E           AssertionError

onlinejudge/service/atcoder.py:1158: AssertionError
```

--- a/onlinejudge/service/atcoder.py
+++ b/onlinejudge/service/atcoder.py
@@ -1044,6 +1044,11 @@ class AtCoderSubmissionData(SubmissionData):
 
 
 class AtCoderSubmissionDetailedData(AtCoderSubmissionData):
+    """
+    .. deprecated:: 10.5
+       The feature has been broken at https://github.com/online-judge-tools/api-client/issues/110
+    """
+
     # yapf: disable
     def __init__(
             self,
@@ -1130,102 +1135,12 @@ class AtCoderSubmission(onlinejudge.type.Submission):
 
     def download_data(self, *, session: Optional[requests.Session] = None) -> AtCoderSubmissionDetailedData:
         """
-        :note: `Exec Time` is undefined when the status is `RE` or `TLE`
-        :note: `Memory` is undefined when the status is `RE` or `TLE`
+        :raises NotImplementedError:
+
+        .. deprecated:: 10.5
+           The feature has been broken at https://github.com/online-judge-tools/api-client/issues/110
         """
-        session = session or utils.get_default_session()
-        resp = _request('GET', self.get_url(type='beta', lang='en'), session=session)
-        soup = bs4.BeautifulSoup(resp.content.decode(resp.encoding), utils.html_parser)
-        timestamp = datetime.datetime.now(datetime.timezone.utc).astimezone()
-
-        # Submission #N
-        id_, = soup.find_all('span', class_='h2')
-        assert id_.text == 'Submission #{}'.format(self.submission_id)
-
-        # Source Code
-        source_code = soup.find(id='submission-code')
-        source_code = source_code.text.encode()
-
-        # get tables
-        tables = soup.find_all('table')
-        if len(tables) == 3:
-            submission_info, test_cases_summary, test_cases_data = tables
-        elif len(tables) == 1:
-            submission_info, = tables
-            test_cases_summary = None
-            test_cases_data = None
-        else:
-            assert False
-
-        # Submission Info
-        data = {}  # type: Dict[str, str]
-        problem_id = None  # type: Optional[str]
-        for tr in submission_info.find_all('tr'):
-            key = tr.find('th').text.strip()
-            value = tr.find('td').text.strip()
-            data[key] = value
-
-            if key == 'Task':
-                problem = AtCoderProblem.from_url('https://atcoder.jp' + tr.find('a')['href'])
-                assert problem is not None
-                problem_id = problem.problem_id
-
-        assert problem_id is not None
-        submission_time = datetime.datetime.strptime(data['Submission Time'], '%Y-%m-%d %H:%M:%S+0900').replace(tzinfo=utils.tzinfo_jst)
-        user_id = data['User']
-        language_name = data['Language']
-        score = float(data['Score'])
-        code_size = int(utils.remove_suffix(data['Code Size'], ' Byte'))
-        status = data['Status']
-        if 'Exec Time' in data:
-            exec_time_msec = int(utils.remove_suffix(data['Exec Time'], ' ms'))  # type: Optional[int]
-        else:
-            exec_time_msec = None
-        if 'Memory' in data:
-            # TODO: confirm this is KB truly, not KiB
-            memory_byte = int(utils.remove_suffix(data['Memory'], ' KB')) * 1000  # type: Optional[int]
-        else:
-            memory_byte = None
-
-        # Compile Error
-        compile_error_tag = soup.find('h4', text='Compile Error')
-        if compile_error_tag is not None:
-            compile_error = compile_error_tag.find_next_sibling('pre').text
-        else:
-            compile_error = None
-
-        # Test Cases
-        if test_cases_summary is not None:
-            trs = test_cases_summary.find('tbody').find_all('tr')
-            test_sets = [AtCoderSubmissionTestSet._from_table_row(tr) for tr in trs]  # type: Optional[List[AtCoderSubmissionTestSet]]
-        else:
-            test_sets = None
-        if test_cases_data is not None:
-            trs = test_cases_data.find('tbody').find_all('tr')
-            test_cases = [AtCoderSubmissionTestCaseResult._from_table_row(tr) for tr in trs]  # type: Optional[List[AtCoderSubmissionTestCaseResult]]
-        else:
-            test_cases = None
-
-        return AtCoderSubmissionDetailedData(
-            code_size=code_size,
-            compile_error=compile_error,
-            exec_time_msec=exec_time_msec,
-            language_name=language_name,
-            memory_byte=memory_byte,
-            problem=AtCoderProblem(contest_id=self.contest_id, problem_id=problem_id),
-            problem_id=problem_id,
-            response=resp,
-            score=score,
-            session=session,
-            source_code=source_code,
-            status=status,
-            submission=self,
-            submission_time=submission_time,
-            test_cases=test_cases,
-            test_sets=test_sets,
-            timestamp=timestamp,
-            user_id=user_id,
-        )
+        raise NotImplementedError
 
 
 class AtCoderSubmissionTestSet:
@@ -1234,6 +1149,9 @@ class AtCoderSubmissionTestSet:
     :ivar score: :py:class:`float`
     :ivar max_score: :py:class:`float`
     :ivar test_case_names: :py:class:`List` [ :py:class:`str` ]
+
+    .. deprecated:: 10.5
+       The feature has been broken at https://github.com/online-judge-tools/api-client/issues/110
     """
     def __init__(self, *, set_name: str, score: float, max_score: float, test_case_names: List[str]):
         self.set_name = set_name
@@ -1243,12 +1161,10 @@ class AtCoderSubmissionTestSet:
 
     @classmethod
     def _from_table_row(cls, tr: bs4.Tag) -> 'AtCoderSubmissionTestSet':
-        tds = tr.find_all('td')
-        assert len(tds) == 3
-        set_name = tds[0].text
-        score, max_score = [float(s) for s in tds[1].text.split('/')]
-        test_case_names = tds[2].text.split(', ')
-        return AtCoderSubmissionTestSet(set_name=set_name, score=score, max_score=max_score, test_case_names=test_case_names)
+        """
+        :raises NotImplementedError:
+        """
+        raise NotImplementedError
 
 
 class AtCoderSubmissionTestCaseResult:
@@ -1257,6 +1173,9 @@ class AtCoderSubmissionTestCaseResult:
     :ivar status: :py:class:`str`
     :ivar exec_time_msec: :py:class:`int` in millisecond
     :ivar memory_byte: :py:class:`int` in byte
+
+    .. deprecated:: 10.5
+       The feature has been broken at https://github.com/online-judge-tools/api-client/issues/110
     """
     def __init__(self, *, case_name: str, status: str, exec_time_msec: Optional[int], memory_byte: Optional[int]):
         self.case_name = case_name
@@ -1266,17 +1185,10 @@ class AtCoderSubmissionTestCaseResult:
 
     @classmethod
     def _from_table_row(cls, tr: bs4.Tag) -> 'AtCoderSubmissionTestCaseResult':
-        tds = tr.find_all('td')
-        case_name = tds[0].text
-        status = tds[1].text
-        exec_time_msec = None  # type: Optional[int]
-        memory_byte = None  # type: Optional[int]
-        if len(tds) == 4:
-            exec_time_msec = int(utils.remove_suffix(tds[2].text, ' ms'))
-            memory_byte = int(utils.remove_suffix(tds[3].text, ' KB')) * 1000  # TODO: confirm this is KB truly, not KiB
-        else:
-            assert len(tds) == 2
-        return AtCoderSubmissionTestCaseResult(case_name=case_name, status=status, exec_time_msec=exec_time_msec, memory_byte=memory_byte)
+        """
+        :raises NotImplementedError:
+        """
+        raise NotImplementedError
 
 
 onlinejudge.dispatch.services += [AtCoderService]

--- a/tests/dispatch.py
+++ b/tests/dispatch.py
@@ -17,9 +17,7 @@ class DispatchAtCoderTest(unittest.TestCase):
         submission = dispatch.submission_from_url('https://atcoder.jp/contests/agc039/submissions/7874055')
         self.assertIsInstance(submission, service.atcoder.AtCoderSubmission)
         self.assertIsInstance(submission.get_service(), service.atcoder.AtCoderService)
-        problem = submission.download_problem()
-        self.assertEqual(problem.contest_id, "agc039")
-        self.assertEqual(problem.problem_id, "agc039_b")
+        self.assertEqual(submission.submission_id, 7874055)
 
     def test_contest_from_url(self):
         contest = dispatch.contest_from_url('https://atcoder.jp/contests/agc030')

--- a/tests/service_atcoder.py
+++ b/tests/service_atcoder.py
@@ -88,9 +88,9 @@ class AtCoderContestTest(unittest.TestCase):
         contest = AtCoderContest.from_url('https://atcoder.jp/contests/dwacon2018-final-open')
         problems = contest.list_problems()
         self.assertEqual(problems[0].download_data().time_limit_msec, 2525)
-        self.assertEqual(problems[0].download_data().memory_limit_byte, int(252.525 * 1000 * 1000))
+        self.assertEqual(problems[0].download_data().memory_limit_byte, 246 * 1000 * 1000)
         self.assertEqual(problems[1].download_data().time_limit_msec, 5252)
-        self.assertEqual(problems[1].download_data().memory_limit_byte, int(525.252 * 1000 * 1000))
+        self.assertEqual(problems[1].download_data().memory_limit_byte, 512 * 1000 * 1000)
 
     def test_list_problems_time_limit_is_less_than_msec(self):
         contest = AtCoderContest.from_url('https://atcoder.jp/contests/joi2019ho')
@@ -214,7 +214,7 @@ class AtCoderProblemDataTest(unittest.TestCase):
         self.assertEqual(data.available_languages, None)
         self.assertEqual(data.html, html)
         self.assertEqual(data.input_format, None)
-        self.assertEqual(data.memory_limit_byte, 300 * 1000 * 1000)
+        self.assertEqual(data.memory_limit_byte, 292 * 1000 * 1000)
         self.assertEqual(data.name, 'プログラミングコンテスト')
         self.assertEqual(data.problem, AtCoderProblem.from_url(url))
         self.assertEqual(data.sample_cases, [

--- a/tests/service_atcoder.py
+++ b/tests/service_atcoder.py
@@ -111,10 +111,8 @@ class AtCoderContestTest(unittest.TestCase):
         contest = AtCoderContest.from_url('https://atcoder.jp/contests/code-festival-2014-exhibition-open')
         submissions = list(contest.iterate_submissions())
         self.assertGreater(len(submissions), 300)
-        self.assertEqual(submissions[0].download_data().code_size, 276)
-        self.assertEqual(submissions[0].download_data().status, 'WA')
-        self.assertEqual(submissions[1].download_data().user_id, 'snuke')
-        self.assertEqual(submissions[1].download_data().status, 'WA')
+        self.assertEqual(submissions[0].get_url(), 'https://atcoder.jp/contests/code-festival-2014-exhibition-open/submissions/272697')
+        self.assertEqual(submissions[1].get_url(), 'https://atcoder.jp/contests/code-festival-2014-exhibition-open/submissions/272700')
 
     def test_get_contest_without_penalty(self):
         contest = AtCoderContest.from_url('https://atcoder.jp/contests/otemae2019')
@@ -191,10 +189,10 @@ class AtCoderProblemTest(unittest.TestCase):
     def test_iterate_submissions(self):
         problem = AtCoderProblem.from_url('https://atcoder.jp/contests/abc119/tasks/abc119_c')
         submissions = problem.iterate_submissions()
-        self.assertEqual(next(submissions).download_data().score, 300)
-        self.assertEqual(next(submissions).download_data().code_size, 1208)
-        self.assertEqual(next(submissions).download_data().exec_time_msec, 2)
-        self.assertEqual(next(submissions).download_data().memory_byte, 256 * 1000)
+        self.assertEqual(next(submissions).get_url(), 'https://atcoder.jp/contests/abc119/submissions/4368719')
+        self.assertEqual(next(submissions).get_url(), 'https://atcoder.jp/contests/abc119/submissions/4368922')
+        self.assertEqual(next(submissions).get_url(), 'https://atcoder.jp/contests/abc119/submissions/4369188')
+        self.assertEqual(next(submissions).get_url(), 'https://atcoder.jp/contests/abc119/submissions/4369193')
 
 
 class AtCoderSubmissionTest(unittest.TestCase):
@@ -203,98 +201,6 @@ class AtCoderSubmissionTest(unittest.TestCase):
         self.assertEqual(AtCoderSubmission.from_url('https://atcoder.jp/contests/kupc2012/submissions/2097011').submission_id, 2097011)
         self.assertEqual(AtCoderSubmission.from_url('https://qupc2014.contest.atcoder.jp/submissions/1444440').contest_id, 'qupc2014')
         self.assertEqual(AtCoderSubmission.from_url('https://qupc2014.contest.atcoder.jp/submissions/1444440').submission_id, 1444440)
-
-    def test_submission_info(self):
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/agc030/submissions/3904911')
-        data = submission.download_data()
-        self.assertEqual(data.submission_time.year, 2018)
-        self.assertEqual(data.submission_time.month, 12)
-        self.assertEqual(data.submission_time.day, 31)
-        self.assertEqual(data.user_id, 'kimiyuki')
-        self.assertEqual(data.problem.problem_id, 'agc030_b')
-        self.assertEqual(data.language_name, 'C++14 (GCC 5.4.1)')
-        self.assertEqual(data.score, 800)
-        self.assertEqual(data.code_size, 1457)
-        self.assertEqual(data.status, 'AC')
-        self.assertEqual(data.exec_time_msec, 85)
-        self.assertEqual(data.memory_byte, 3328 * 1000)
-
-    def test_submission_info_compile_error(self):
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/abc124/submissions/4943518')
-        data = submission.download_data()
-        self.assertEqual(data.submission_time.year, 2019)
-        self.assertEqual(data.submission_time.month, 4)
-        self.assertEqual(data.submission_time.day, 13)
-        self.assertEqual(data.user_id, 'pekempey')
-        self.assertEqual(data.problem.problem_id, 'abc124_d')
-        self.assertEqual(data.language_name, 'Rust (1.15.1)')
-        self.assertEqual(data.score, 0)
-        self.assertEqual(data.code_size, 787)
-        self.assertEqual(data.status, 'CE')
-        self.assertEqual(data.exec_time_msec, None)
-        self.assertEqual(data.memory_byte, None)
-
-    def test_submission_info_compile_warnings(self):
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/agc032/submissions/4675493')
-        data = submission.download_data()
-        self.assertEqual(data.submission_time.year, 2019)
-        self.assertEqual(data.submission_time.month, 3)
-        self.assertEqual(data.submission_time.day, 23)
-        self.assertEqual(data.user_id, 'yutaka1999')
-        self.assertEqual(data.problem.problem_id, 'agc032_e')
-        self.assertEqual(data.language_name, 'C++14 (GCC 5.4.1)')
-        self.assertEqual(data.score, 0)
-        self.assertEqual(data.code_size, 1682)
-        self.assertEqual(data.status, 'WA')
-        self.assertEqual(data.exec_time_msec, 392)
-        self.assertEqual(data.memory_byte, 7168 * 1000)
-
-    def test_get_test_sets(self):
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/arc028/submissions/223928')
-        test_cases = submission.download_data().test_sets
-        self.assertEqual(len(test_cases), 3)
-        self.assertEqual(test_cases[0].set_name, 'Sample')
-        self.assertEqual(test_cases[0].score, 0)
-        self.assertEqual(test_cases[0].max_score, 0)
-        self.assertEqual(test_cases[0].test_case_names, ['sample_01.txt', 'sample_02.txt'])
-        self.assertEqual(test_cases[1].set_name, 'Subtask1')
-        self.assertEqual(test_cases[1].score, 40)
-        self.assertEqual(test_cases[1].max_score, 40)
-        self.assertEqual(len(test_cases[1].test_case_names), 13)
-        self.assertEqual(test_cases[2].set_name, 'Subtask2')
-        self.assertEqual(test_cases[2].score, 0)
-        self.assertEqual(test_cases[2].max_score, 60)
-        self.assertEqual(len(test_cases[2].test_case_names), 20)
-
-    def test_get_test_cases(self):
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/tricky/submissions/119944')
-        test_cases = submission.download_data().test_cases
-        self.assertEqual(len(test_cases), 2)
-        self.assertEqual(test_cases[0].case_name, 'input_01.txt')
-        self.assertEqual(test_cases[0].status, 'TLE')
-        self.assertEqual(test_cases[0].exec_time_msec, 2031)
-        self.assertEqual(test_cases[0].memory_byte, 9220000)
-        self.assertEqual(test_cases[1].case_name, 'input_02.txt')
-        self.assertEqual(test_cases[1].status, 'AC')
-        self.assertEqual(test_cases[1].exec_time_msec, 131)
-        self.assertEqual(test_cases[1].memory_byte, 7400 * 1000)
-
-    def test_get_source_code(self):
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/abc100/submissions/3082514')
-        self.assertEqual(submission.download_data().source_code, b'/9\\|\\B/c:(\ncYay!')
-        self.assertEqual(submission.download_data().code_size, 16)
-
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/abc100/submissions/4069980')
-        self.assertEqual(submission.download_data().source_code, b'/9\\|\\B/c:(\r\ncYay!')
-        self.assertEqual(submission.download_data().code_size, 17)
-
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/abc100/submissions/4317534')
-        self.assertEqual(submission.download_data().source_code, b'/9\\|\\B/c:(\r\ncYay!\r\n')
-        self.assertEqual(submission.download_data().code_size, 19)
-
-    def test_get_score_float(self):
-        submission = AtCoderSubmission.from_url('https://atcoder.jp/contests/pakencamp-2018-day3/submissions/4583531')
-        self.assertAlmostEqual(submission.download_data().score, 32.53)
 
 
 class AtCoderProblemDataTest(unittest.TestCase):

--- a/tests/type.py
+++ b/tests/type.py
@@ -1,7 +1,6 @@
 import unittest
 
 from onlinejudge.dispatch import problem_from_url, service_from_url, submission_from_url
-from onlinejudge.service import atcoder
 from onlinejudge.type import Problem, Service, Submission
 
 
@@ -23,9 +22,3 @@ class TypeTest(unittest.TestCase):
     def test_submission_eq(self):
         self.assertEqual(submission_from_url('https://atcoder.jp/contests/abc143/submissions/8264863'), submission_from_url('https://atcoder.jp/contests/abc143/submissions/8264863'))
         self.assertNotEqual(submission_from_url('https://atcoder.jp/contests/abc143/submissions/8264863'), submission_from_url('https://atcoder.jp/contests/abc143/submissions/8264897'))
-
-    def test_download_contest(self):
-        submission = submission_from_url('https://atcoder.jp/contests/abc143/submissions/8059104')
-        contest = submission.download_contest()
-        self.assertIsInstance(contest, atcoder.AtCoderContest)
-        self.assertEqual(contest.contest_id, 'abc143')

